### DISCUSSION TITLE
Issue with adding a mod

### DISCUSSION THREAD

**Discussion by AvallonAB** [gorilla-devs/ferium#348] [Q&A] [answered]
I have an issue with adding a mod from [this](https://github.com/19MisterX98/SeedcrackerX) github repostory. The command I'm using is `ferium add 19MisterX98/SeedcrackerX`. I don't know what appears to be the issue here since the files in releases are .jar files. I'm sorry if this is something dumb, I'm new to this kind of stuff.

edit: I don't know if I should make a separate threat for that or not but can you disable version checks only when adding a mod? Some mods that I use aren't updated for the newest MC version yet but I still want to add them without ferium downloading it for an older version (I know it's possible via editing config after adding the mod with the --dont-check-game-version but that can be kinda tedious)

**Comment by theRookieCoder** [comment, marked as answer]
The repository doesn't specify the mod loader (as it is assumed to be Fabric) so you should add `--dont-check-mod-loader`. Also it seems the game version isn't specified in the filename, so unfortunately this author's release format just won't work with ferium. If you're fine with downloading the latest version everytime, then `ferium add 19MisterX98/SeedcrackerX --dont-check-mod-loader --dont-check-game-version` could work, but it's probably not gonna work well, it might be better to just manually download the mod in this case.

**Reply by AvallonAB** [reply]
thank you!
